probe-rs/src/probe/espusbjtag/protocol.rs

```diff
@@ -309,6 +288,13 @@ impl ProtocolHandler {
             }
         }
 
+        Ok(())
+    }
+
+    /// Flushes all the pending commands to the JTAG adapter.
+    pub fn read_captured_bits(&mut self) -> Result<BitVec<u8, Lsb0>, DebugProbeError> {
```

Comment:
No we have two methods with the same description :D I guess one just takes the response and one does not?

Reply:
oops